netplugin/netd.go

```diff
@@ -76,6 +77,11 @@ func startNetPlugin(pluginConfig *plugin.Config) {
 }
 
 func initNetPluginConifg(ctx *cli.Context) (plugin.Config, error) {
+	// Make sure we are running as root
+	uid := syscall.Getuid()
+	if uid != 0 {
+		log.Fatalf("This process can only be run as root")
+	}
```

Comment:
is this required to talk to ovs?  though wei removed this because it was not required

Reply:
@chrisplo: I can remove it if it's not needed.